src/dom/nodes/CdataSectionRe.re

```diff
@@ -1,43 +1,8 @@
 type t = Dom.cdataSection;
 
-include
-  EventTargetRe.Impl(
-    {
-      type nonrec t = t;
-    }
-  );
-
-include
-  NodeRe.Impl(
-    {
-      type nonrec t = t;
-    }
-  );
-
-include
-  CharacterDataRe.Impl(
-    {
-      type nonrec t = t;
-    }
-  );
-
-include
-  NonDocumentTypeChildNodeRe.Impl(
-    {
-      type nonrec t = t;
-    }
-  );
-
-include
```

Comment:
While toplevel modules get one-lined, One-line functor arguments apparently need to be exploded into six lines, lol.